src/codecs/webp/extended.rs

```diff
@@ -633,7 +674,7 @@ pub(crate) fn read_extended_header<R: Read>(reader: &mut R) -> ImageResult<WebPE
 }
 
 fn read_anim_frame<R: Read>(
-    mut reader: R,
+    mut reader: &mut R,
```

Comment:
This change shouldn't be needed since there is a blanket `impl<R: Read> Read for &mut R` 😄